packages/lib/services/plugins/defaultPlugins/README.md

```diff
@@ -0,0 +1,33 @@
+## How to add a new default plugin
+ 
+To add a new default plugin for desktop,
+- Add the plugin ID and pinned version to `defaultPlugins` object located [here](https://github.com/laurent22/joplin/blob/dev/packages/lib/services/plugins/defaultPlugins/desktopDefaultPluginsInfo.ts#L5).
+- If necessary, you can also add default settings for the plugins. 
+
+For example, if you wanted to add 2 default settings, `settingName1` and `settingName2` then you will modify the `defaultPlugins` object in following way:
```

Comment:
Make a child item of the above list item?

`,` after `settingName2`

Reply:
Done :+1: